src/utils/filter-utils.js

```diff
@@ -59,6 +58,15 @@ export const PLOT_TYPES = keyMirror({
   lineChart: null
 });
 
+export const FILTER_UPDATER_PROPS = keyMirror({
+  dataId: null,
+  name: null
+});
+
+export const LIMITED_FILTER_EFFECT_PROPS = keyMirror({
```

Comment:
For this list of properties we can apply filter side effects to only one dataset